core/trino-main/src/main/java/io/trino/sql/planner/iterative/rule/PruneDistinctAggregation.java

```diff
@@ -128,8 +127,7 @@ public PlanNode visitAggregation(AggregationNode node, Boolean context)
         {
             boolean distinct = isDistinctOperator(node);
 
-            PlanNode rewrittenNode = getOnlyElement(lookup.resolveGroup(node.getSource())
-                    .map(source -> source.accept(this, distinct)).collect(Collectors.toList()));
+            PlanNode rewrittenNode = lookup.resolve(node.getSource()).accept(this, distinct);
```

Comment:
Separate this from "Replace invalid resolveGroup usage with resolve" commit.

This wasn't incorrect. That was just overly verbose, i.e. using `resolveGroup` without real group capability

Reply:
Moved to "Deduplicate Lookup.resolve copies" commit